### PR TITLE
Remove submission system from Architecture and streamline first paras

### PR DESCRIPTION
**Pull request by hpk42**
also refine README a little.

this is not a complete edit but i think it's worthwhile to merge this already.

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -55,31 +42,31 @@ app-upgrading requests and requests which initially add a new webxdc to the stor
...
 ## Updating the App Index
-Updating the App Index should work in two ways:
 
-1. By importing newer versions with the CLI.
-2. By updating them from a `Submit` chat.
+Updating the App Index happens by importing newer versions of webxdc app files with the CLI. 
+For every added webxdc app the `app-id` and `version` are checked and a new `AppInfo` is created 
+and added to the database. 
+If there already is an `AppInfo` with the same `app-id` but an older version, 
+then this older `AppInfo` will be invalidated.
```
[the remark this answers is not in the document]
Comment (hpk42) [reply ARCHITECTURE.md]: i changed this -- not sure it's exactlyu right -- if not, please provide a suggestion.

File: README.md [hunk 2]
```diff
@@ -5,17 +5,20 @@
...
 
 - The store bot command line interface allows to import apps to 
   make them available in the **app index** which is presented by the store xdc. 
 
+Currently all Bot/Frontend communication happens via Delta Chat messaging. 
+The Bot does not need to be internet-reachable. 
 
-## Getting started with sharing a first app to a chat group 
+
+## Getting started with sharing ("installing") a first app to a chat group
```
Comment (adbenitez) [inline README.md:21]: not sure about introducing this "installing" term here

File: ARCHITECTURE.md [hunk 3]
```diff
@@ -1,51 +1,38 @@
...
-  When this chat is created, the original submitter is added to the chat, 
-  the submitted webxdc is forwarded and a helper xdc (submit-helper.xdc) is sent to the chat.
+The Bot maintains 1:1 chats with each of its users 
+who can get in contact via a QR code or by sending an arbitrary message. 
 
-- The `Review` chat is created by clicking the submit button in the `submit-helper.xdc` (previous step). 
-  When creating such a chat, the bot chooses at most 3 testers and one publisher and adds them to the chat 
-  together with one forwarded instance of the submitted webxdc as well as the `review-helper.xdc`. 
-  When all tests are complete, the publisher can click on `publish` to finally release the webxdc to the store.
+The bot currently uses two different chat types: `Store` and `Genesis`:
```
Comment (adbenitez) [inline ARCHITECTURE.md:7]: is this "Genesis" still applicable? IIRC the submission part got removed

Comment (hpk42) [reply ARCHITECTURE.md:7]: genesis is still there but not doing anything.

**Review by Septias — APPROVED** [review]

**Pull request merged**